okta/resource_okta_user_base_schema_test.go

```diff
@@ -13,11 +14,21 @@ import (
 const baseTestProp = "firstName"
 
 func sweepUserBaseSchema(client *testClient) error {
-	_, _, err := client.apiSupplement.GetUserSchema()
+	var errorList []error
+	schemaUrl := "/api/v1/meta/schemas/user/default"
+
+	schema, _, err := client.apiSupplement.GetUserSchema(schemaUrl)
```

Comment:
@bogdanprodan-okta This is just a note, Okta SDK should handle this and not the extend SDK.

Reply:
Yes, but currently official SDK doesn't support this endpoint, thus internal SDK is used.